data/json/items/melee.json

```diff
@@ -3805,6 +3805,31 @@
     "use_action": { "type": "knife" },
     "category" : "weapons",
     "qualities": [["CUT", 1], ["BUTCHER", 8]]
+    },{
+    "id": "scimitar",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "scimitar",
+    "description": "This is a curved sword associated with various middle eastern and central asian countries. Designed for slashing, and quite deadly against unarmored targets.",
```

Comment:
Ack. I should've remembered to do that when it was mentioned in the big mainlining PR. @_@